apps/demos/Demos/DropDownButton/Overview/React/DropDownButtonTemplate.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Avatar = ({ icon }) => (
+  <div className="button-img-container">
+    <div className="button-img-indicator"></div>
+    <img className="button-img" src={icon} alt="employee" />
+  </div>
+);
+
+const ButtonDescription = ({ text, description }) => (
+  <div className="text-container">
+    <div className="button-title">{text}</div>
+    <div className="button-row">{description}</div>
+  </div>
+);
+
+const DropDownButtonTemplate = () => (
```

Comment:
```suggestion
export default DropDownButtonTemplate = () => (
```

Reply:
I think template doesn't render properly when we try to change const to export default.